_posts/2024-04-09-strimzicon2024-schedule.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title:  "StrimziCon 2024: Schedule announced!"
+date: 2024-04-09
+author: paolo_patierno
+---
+
+We are very pleased to announce the speakers and sessions for the first StrimziCon!
+The interest from the community was really high and we received a lot of awesome proposals.
+The program committee had really tough times to choose among them in order to build an amazing agenda.
+
+<!--more-->
+
+![StrimziCon 2024 Banner](/assets/images/posts/2024-01-29-strimzicon2024-banner.png)
+
+### Awesome sessions ... join the StrimziCon! 
+
+The conference will take place on <b>Wednesday, May 22nd</b> (afternoon CEST), with two parallel tracks in the "River" and "Waterfall" virtual rooms.
+It will start with a keynote with Kate Stanley (Red Hat) and Paolo Patierno (Red Hat, Strimzi maintainer) talking about Strimzi, its current status and the future roadmap together with the Apache Kafka project.
```

Comment:
```suggestion
It will start with a keynote from Kate Stanley (Red Hat) and Paolo Patierno (Red Hat, Strimzi maintainer) talking about Strimzi, its current status and the future roadmap together with the Apache Kafka project.
```